db/migrate/20170911205637_add_indexes.rb

```diff
@@ -0,0 +1,11 @@
+class AddIndexes < ActiveRecord::Migration[5.1]
+  def change
+    add_index :shelters, :active
+    add_index :shelters, :accepting
+    add_index :shelters, :latitude
```

Comment:
I think the indexes on latitude and longitude should be a composite index.

https://github.com/alexreisner/geocoder#optimisation-of-distance-queries